src/devices/Ws28xx/README.md

```diff
@@ -66,7 +67,8 @@ void Rainbow(Ws28xx neo, int count, int iterations = 1)
     }
 }
 ```
-
+#### Note: 
+Using the SK6812 is almost the same, but the alpha channel of the color is used for the white LED. This means that the predefined color definitions (like ```System.Drawing.Color.Red``` will not work correctly as they have the alpha channel set to 255 (0xFF). That will turn the white LED always on. See the [sample](samples/SK6812_Samples/Programs.cs) for the main differences to the above code.
```

Comment:
please add an extra enter after and before the title. In Markdown, each title has to have an extra enter before and after it.

Reply:
To be more explicit as in the sample it's not that obvious, can you add a code snippet here to show how to handle the color management?